package.json

```
@@ -18,6 +18,7 @@
     "test_unit": "gulp test:scripts:unit:watch",
     "test_unit_no_watch": "gulp test:scripts:unit",
     "test_functional": "chimp tests/functional/chimp.js",
+    "test_census": "./node_modules/cypress/bin/cypress open",
```

Also don't need to point to node_modules within `package.json`. 
```suggestion
    "test_census": "cypress run --spec \"tests/functional/cypress/spec/census-household.spec.js\"",
```

And if we just wanted to run all the cy tests, we can just do run.
```suggestion
    "test_census": "cypress run",
```

---

Can we also have an option to not run it headless?

---

Used the current env var to do this as per @MebinAbraham's suggestion